### ISSUE TITLE
`indent`: Wrong indentation in template literal expressions

### ISSUE THREAD

**Issue by remziatay** [eslint-stylistic/eslint-stylistic#551]
### Validations

- [X] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

Indentation seems to be off in template literal expressions in some cases. This was working correctly on version 2.6.3 (or maybe it didn't do any formatting checks at all)

## Config options
```js
stylistic.configs.customize({
  indent: 2,
  quotes: 'single',
  semi: true,
  jsx: false,
  braceStyle: '1tbs',
}),
```

## Expected Behavior
Since the plugin won't parse the contents in template literal, I think expected behaviour would be not checking it at all (as on version 2.6.3) and leave as is. This could also be optional

```ts
`
  <div>
    <div>
      <div>
        <article
          test="test"
          class=${clsx('container', {
            hide: 'shouldHide',
            empty: 'isCollapsed',
          })}
        >
          <div class="header"></div>
        </article>
      </div>
    </div>
  </div>
`;
```

## Actual Behavior
Automatically moves indentation of expression to the first line of literal

```ts
`
  <div>
    <div>
      <div>
        <article
          test="test"
          class=${clsx('container', {
    hide: 'shouldHide',
    empty: 'isCollapsed',
  })}
        >
          <div class="header"></div>
        </article>
      </div>
    </div>
  </div>
`;
```

### Reproduction

https://stackblitz.com/edit/github-2gg16p

### Contributes

- [x] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by remziatay** [comment]
Upon further inspection, it looks like the problem occurs when the whole line doesn't start with `${`

This is linted correctly
```ts
`
  <div>
    <div>
      <div>
        <article
          test="test"
          class=
          ${
            clsx('container', {
              hide: 'shouldHide',
              empty: 'isCollapsed',
            })
          }
        >
          <div class="header"></div>
        </article>
      </div>
    </div>
  </div>
`;
```
